presc/report/resources/_config.yml

```diff
@@ -5,6 +5,7 @@ only_build_toc_files: true
 # Force re-execution of notebooks on each build.
 execute:
   execute_notebooks: force
+  timeout: 300
 
```

Comment:
You noticed a timeout issue when running a report of this size?

Reply:
The default timeout was 30 seconds, and that tripped on one of the test cases. I thought bump this up to give some more leeway.